tests/preprocessing/test_positionfixes.py

```diff
@@ -87,7 +89,7 @@ def test_generate_staypoints_dtype_consistent(self):
                                                              dist_threshold=25,
                                                              time_threshold=5 * 60)
         assert pfs['user_id'].dtype == stps['user_id'].dtype
-        assert pfs['staypoint_id'].dtype == "float"
+        assert pfs['staypoint_id'].dtype == "Int64"
         assert stps.index.dtype == "int64"
```

Comment:
It seems weird on the first glance to have two different datatypes but it is probably the right thing to do, as there are no missing values in the index.
We just have to be careful if we evaluate index.dtype == dtype